README.md

```diff
@@ -27,7 +27,7 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/220842844-80dcc03f-5d2c-4fdf-8506-eb54a0cf3b9d.png)

**[Optional]**

- [ ] It would be nice if you make all your commits Capitalize to make your pull request professional :+1: and to follow the requirements :blush: